### PR TITLE
add serialColumn to BareMetalHost list

### PR DESCRIPTION
**Pull request by suomiy**
@jtomasek @honza

### CODE REVIEW

**Comment by suomiy** [comment]
aa

File: frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostsTable.tsx [hunk 1]
```diff
@@ -103,6 +111,7 @@ const HostsTableRow: React.FC<HostsTableRowProps> = ({
         <BareMetalHostRole machine={machine} node={node} />
       </TableData>
       <TableData className={tableColumnClasses.address}>{address}</TableData>
+      <TableData className={tableColumnClasses.serialNumber}>{serialNumber}</TableData>
```
Comment (jtomasek) [inline frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostsTable.tsx]:
```suggestion
      <TableData className={tableColumnClasses.serialNumber}>{serialNumber || DASH}</TableData>
```

Comment (suomiy) [reply frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostsTable.tsx]: done, also added the `DASH` to `address`

**Pull request merged**